### PR TITLE
Created filled progress bar

### PR DESCRIPTION
**Pull request by ewelina-slepko**

### CODE REVIEW

File: src/app/home/layouts/budgets-form/budgets-form.component.ts [hunk 1]
```diff
@@ -7,9 +8,14 @@ import { Component, OnInit } from '@angular/core';
...
+  constructor(private initialSettingsService: InitialSettingsService) {
+  }
 
-  ngOnInit(): void {
+  ngOnInit() {
+    this.sendCurrentStepInfo();
   }
 
+  sendCurrentStepInfo() {
+    this.initialSettingsService.sendCurrentStepInfo('step3');
```
Comment (Rementh) [inline src/app/home/layouts/budgets-form/budgets-form.component.ts]: Is there a need for keeping the step number as a string? I think simple number would be just enough for that.

File: src/app/home/layouts/wallet-form/wallet-form.component.ts [hunk 2]
```diff
@@ -12,29 +13,39 @@ export class WalletFormComponent implements OnInit {
...
     this.sources.push({
       id: ++this.sourceNumber,
       name: 'Card',
       isEditMode: false
     });
   }
+
+  saveBalance(form) {
+    if (form.form.status === 'VALID') {
+      console.log(form.form.value);
```
Comment (Rementh) [inline src/app/home/layouts/wallet-form/wallet-form.component.ts]: konsol

**Comment by ewelina-slepko** [comment]
Done!

**Comment by ewelina-slepko** [comment]
Done!

**Pull request merged**